cross/numpy/Makefile

```diff
@@ -19,3 +19,8 @@ ADDITIONAL_CFLAGS += -O0
 endif
 
 include ../../mk/spksrc.python-wheel.mk
+
+ifeq ($(PYTHON_LIB_NATIVE),$(PYTHON_LIB_CROSS))
+LD_LIBRARY_PATH := $(INSTALL_DIR)$(INSTALL_PREFIX)/lib:$(LD_LIBRARY_PATH)
```

Comment:
When running numpy build, cython code is actually executed. The issue here is that, when not cross compiling, binary code was executed and required python libraries weren't in the path.